### ISSUE TITLE
Create by PUT inside transaction doesn't work

### ISSUE THREAD

**Issue by draakhan** [fhirbase/fhirbase-plv8#123]
Hi!

Issue found on v1.3.0.15. When inside transaction resource with given id is to be created by PUT, fhirbase raises error and resource isn't created.

```
fhirbase=# SELECT fhir_drop_storage('{"resourceType": "Patient"}');
-[ RECORD 1 ]-----+------------------------------------------------------
fhir_drop_storage | {"status":"ok","message":"Table patient was dropped"}

fhirbase=# SELECT fhir_create_storage('{"resourceType": "Patient"}');
-[ RECORD 1 ]-------+------------------------------------------------------
fhir_create_storage | {"status":"ok","message":"Table patient was created"}

fhirbase=# SELECT resource ->> 'name' from patient;
(No rows)
fhirbase=# SELECT resource FROM patient;
(No rows)
fhirbase=# SELECT fhir_transaction('{"resourceType":"Bundle","type":"transaction","entry":[{"resource":{"resourceType":"Patient","id":"bbb-ccc-ddd-123","active":true,"name":[{"use":"official","family":["Snow"],"given":["John"]}],"gender":"male","birthDate":"2001-01-01"},"request":{"method":"PUT","url":"Patient/bbb-ccc-ddd-123"}}]}');
-[ RECORD 1 ]----+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
fhir_transaction | {"resourceType":"OperationOutcome","message":"There were incorrect requests within transaction. [{\"type\":\"error\",\"message\":\"Cannot determine action for request PUT Patient/bbb-ccc-ddd-123\"}]"}

fhirbase=# SELECT resource FROM patient; 
(No rows)
```

**Comment by bartlomiejs-kainos** [comment]
By mistake I tested on different version then v1.3.0.15. I just retested it on correct and the newest version  v1.3.0.21. The problem is the same, but transaction response differs:

```
{"resourceType":"OperationOutcome","issue":[{"severity":"error","code":"422","diagnostics":"Invalid operation PUT Patient/bbb-ccc-ddd-123"}]}
```

**Comment by draakhan** [comment]
By mistake I tested on different version then v1.3.0.15. I just retested it on correct and the newest version  v1.3.0.21. The problem is the same, but transaction response differs:

```
{"resourceType":"OperationOutcome","issue":[{"severity":"error","code":"422","diagnostics":"Invalid operation PUT Patient/bbb-ccc-ddd-123"}]}
```

**Comment by danil** [comment]
@draakhan fixed in v1.3.0.22
https://github.com/fhirbase/fhirbase-plv8/blob/v1.3.0.22/test/integration/issues/123_spec.coffee#L8
if not, feel free to reopen
